src/sources/host_metrics.rs

```diff
@@ -28,13 +28,15 @@ use heim::{
     units::{information::byte, time::second},
     Error,
 };
+#[cfg(target_os = "linux")]
+use heim_runtime;
```

Comment:
What if we move this use to the function itself so it doesn’t have to be kept track of separately?

Reply:
This can actually be completely removed. It's a redundant inclusion.